### PR TITLE
Adds materials

### PR DESCRIPTION
**Pull request by Z-Denis**
This adds abridged body fluids, building materials, carbon allotropes, decorative stones and fabrics

### CODE REVIEW

**Comment by Z-Denis** [comment]
This further adds gemstones, common metals and metals from the periodic table.

File: data/materials/building-materials.json [hunk 1]
```diff
@@ -0,0 +1,285 @@
+{
+  "description": "materiales de construcción",
+  "materiales de construcción": [
+		{
+      "name": "PSL",
```
Comment (xurxodiz) [inline data/materials/building-materials.json]:
```suggestion
                  "name": "PSL",
```

File: data/materials/carbon-allotropes.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    },
+    {
+      "name": "nanoespuma de carbono",
+      "gender": "f",
+      "number": "s"
+    },
+    {
+      "name": "schwarzites",
+      "gender": "m",
+			"number": "p"
```
Comment (xurxodiz) [inline data/materials/carbon-allotropes.json]:
```suggestion
      "number": "p"
```

File: data/materials/decorative-stones.json [hunk 3]
```diff
@@ -0,0 +1,250 @@
+{
+  "description": "piedras decorativas",
+  "piedras decorativas": [
+		{
+      "name": "anortosita",
```
Comment (xurxodiz) [inline data/materials/decorative-stones.json]:
```suggestion
                  "name": "anortosita",
```

File: data/materials/metals.json [hunk 4]
```diff
@@ -0,0 +1,465 @@
...
+    },
+    {
+      "name": "vanadio",
+      "gender": "m",
+      "number": "s"
+    },
+    {
+      "name": "zinc",
+      "gender": "m",
+			"number": "s"
```
Comment (xurxodiz) [inline data/materials/metals.json]:
```suggestion
      "number": "s"
```

File: data/materials/metals.json [hunk 5]
```diff
@@ -0,0 +1,465 @@
...
+		}
```
Comment (xurxodiz) [inline data/materials/metals.json]:
```suggestion
    }
```

**Review by xurxodiz — APPROVED** [review]

**Comment by xurxodiz** [comment]
Wow, this is an incredible effort! Thank you SO MUCH!

**Pull request merged**